ui/helpers/constants/metamask-notifications/metamask-notifications.ts

```diff
@@ -0,0 +1,47 @@
+import { CHAIN_IDS } from '../../../../shared/constants/network';
+
+export type BlockExplorerConfig = {
```

Comment:
Nit: A TSDoc here would be appreciated, it would improve Intellisense

Reply:
Yes, you're right. Done